packages/manager/modules/veeam/src/storage/add/veeam-storage-add.controller.js

```diff
@@ -0,0 +1,21 @@
+export default class VeeamStorageAddCtrl {
+  constructor($uibModalInstance, serviceName, VeeamService) {
```

Comment:
Missing ngInject annotation:

```suggestion
  /* @ngInject */
  constructor($uibModalInstance, serviceName, VeeamService) {
```